lib/utilities/pristine.js

```diff
@@ -85,7 +86,9 @@ function installPristineApp(appName, options) {
 
   chdir(temp.pristinePath);
 
-  var promise = applyCommand('new', appName, extraOptions)
+  var args = extraOptions.concat(['-sb', '-sn', runCommandOptions]);
```

Comment:
Oh I missed the extraOptions setup code above here. I will remove my redundant skips.